### PR TITLE
Changelog for v2.134.0

### PR DESCRIPTION
**Pull request by vEpiphyte**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -4,13 +4,53 @@
...
+  ``storm:model:property``, ``storm:model:tagprop``, and ``storm:model:type``
+  objects.
+  (`#3134 <https://github.com/vertexproject/synapse/pull/3134>`_)
+  (`#3139 <https://github.com/vertexproject/synapse/pull/3139>`_)
 
 Bugfixes
 --------
 - Fix an exception that was raised when setting the parent of a View.
   (`#3131 <https://github.com/vertexproject/synapse/pull/3131>`_)
+  (`#3132 <https://github.com/vertexproject/synapse/pull/3132>`_)
```
Comment (invisig0th) [inline CHANGELOG.rst:47]: Did we miss a line above this one?

**Review by invisig0th — CHANGES_REQUESTED** [review]

File: CHANGELOG.rst [hunk 2]
```diff
@@ -4,13 +4,53 @@
 Synapse Changelog
 *****************
 
-Unreleased - YYYY-MM-DD
-=======================
+v2.134.0 - YYYY-MM-DD
```
Comment (invisig0th) [inline CHANGELOG.rst:7]: should this be dated today?

File: CHANGELOG.rst (hunk 1 above)
Comment (vEpiphyte) [reply CHANGELOG.rst:47]: They were related changes.

File: CHANGELOG.rst [hunk 3]
```diff
@@ -4,13 +4,53 @@
...
+Model Changes
+-------------
+- Updates to the ``risk`` model.
+  (`#3137 <https://github.com/vertexproject/synapse/pull/3137>`_)
+
+  Light Edges
+  -----------
+
+  ``addresses``
+    When used with an ``risk:mitigation`` and an ``ou:technique`` node, the
```
Comment (Cisphyx) [inline CHANGELOG.rst:19]: "a" risk:mitigation

File: CHANGELOG.rst [hunk 4]
```diff
@@ -4,13 +4,53 @@
...
+    edge indicates the mitigation addresses the technique.
+
+Features and Enhancements
+-------------------------
+- Add a ``--forms`` option to the Storm ``scrape`` command. This can be used
+  to limit the forms that are made from scraping the input text. The
+  ``scrape`` command now uses the View scrape interface to generate its
+  matches, which may include scrape functionality added via power-ups.
+  The ``scrape`` command  no longer produces warning messages when matched
```
Comment (Cisphyx) [inline CHANGELOG.rst:28]: Extra space between "command" and "no"

**Pull request merged**